internal/assert/assert.go

```diff
@@ -8,9 +8,9 @@ import (
 	"testing"
 )
 
-// ExpectEqual is a helper function used throughout the unit and integration
+// Equals is a helper function used throughout the unit and integration
 // tests to assert deep equality between an actual and expected value.
-func ExpectEqual(tb testing.TB, actual, expected interface{}) {
+func Equals(tb testing.TB, actual, expected interface{}) {
 	if !reflect.DeepEqual(actual, expected) {
 		_, file, line, _ := runtime.Caller(1)
```

Comment:
You might want to use of tb.Helper instead.

Reply:
Fixed in commit 378d3be.